lib/howl/ui/journal_buffer.moon

```diff
@@ -0,0 +1,51 @@
+import log, signal from howl
+import ActionBuffer from howl.ui
+
+append = table.insert
+
+class JournalBuffer extends ActionBuffer
+  new: =>
+    super!
+
+    @read_only = true
+    @title = 'Howl Journal'
+    @entry_sizes = {}
+
+    signal.connect 'log-entry-appended', (entry) -> @append_entry entry
```

Comment:
There's a garbage collecting issue here, as the signal handlers cause the journal buffers to be pegged. I.e. closed journal buffers are never really removed, and continue to silently process signals invisibly. I'd suggest keeping track of open journal buffers in this file using a weak table, and have on set of signal handlers dispatching to all available buffers.

Reply:
Would it be easier to just save the journal buffer and re-use it when opened again, or just flat-out completely destroy it and disconnect the signals? I can't really imagine a use case where someone would need more than one active at once...

Reply:
Yes, these options would works as well (but if the buffer were to be re-used then signal would need to be disconnected between views as well). As long as there's no penalty when not viewing the journal buffer. I agree that having more than one active doesn't make sense.